examples/material-ui/README.md

```diff
@@ -0,0 +1,15 @@
+# Examples
```

Comment:
@erikras  okay then we don't need to replicate all the examples in material UI. Have I understood correctly ?